docs/howto/manage-charms.md

```diff
@@ -3,9 +3,16 @@
 
 > See first: {external+juju:ref}`Juju | Build a charm <build-a-charm>`, {external+charmcraft:ref}`Charmcraft | Manage charms <manage-charms>`
 
-Hit the ground running with Ops by setting up standard project structure and tools.
+## Prepare your environment
 
-> See more: {external+charmcraft:ref}`Charmcraft | Manage Charmcraft <manage-charmcraft>`
+You'll need the following tools:
+
+- Charmcraft. See {external+charmcraft:ref}`Charmcraft | Manage Charmcraft <manage-charmcraft>`.
+- tox. See the [tox installation instructions](https://tox.wiki/en/stable/installation.html#as-tool).
+
+Instead of installing these tools manually, consider using the `charm-dev` [Multipass](https://canonical.com/multipass) blueprint or [`concierge`](https://github.com/jnsgruk/concierge) to prepare your environment.
```

Comment:
Minor:

I feel that's a bit aspirational.
I used concierge and the defaults are too small, while stepping out of defaults means preparing a configuration YAML by hnad.

Still, I figure these kinks will be ironed out with time.

Reply:
I feel like since Concierge is our responsibility now we should fix it if it's not suitable to be used, and should be encouraging it (for the CI case at least) rather than custom solutions or having people put all the manual steps in their workflows. It is a bit aspirational, but I'd like to encourage people to use it and then fix any issues that arise.